packages/webcert/src/feature/certificate/Inputs/UeCauseOfDeath/UeCauseOfDeathList.tsx

```diff
@@ -90,17 +91,24 @@ const UeCauseOfDeathList: React.FC<Props> = ({ question, disabled }) => {
   }
 
   const handleChange = (value: ValueCauseOfDeath) => {
-    updateList(questionValueList.map((item) => (item.id === value.id ? value : item)))
+    updateList(
+      questionValueList.map((item) => (item.id === value.id ? value : item)),
+      value
+    )
   }
 
-  const updateList = (list: ValueCauseOfDeath[]) => {
+  const updateList = (list: ValueCauseOfDeath[], value?: ValueCauseOfDeath) => {
```

Comment:
I would prefer that date is corrected for every debut in the list on update, no need for extra value parameter for this function.

```typescript
list.map(({...val, debut}) => {
  const date = getValidDate(debut.date)
  return {...val, debut: { ...debut, date: isValid(date) ? date : '' } }
})
```

Reply:
fixed in similar way as in UeCauseOfDeath.tsx, converting Date back to string.